src/compiler/checker.ts

```diff
@@ -1472,6 +1472,9 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     var argumentsSymbol = createSymbol(SymbolFlags.Property, "arguments" as __String);
     var requireSymbol = createSymbol(SymbolFlags.Property, "require" as __String);
     var isolatedModulesLikeFlagName = compilerOptions.verbatimModuleSyntax ? "verbatimModuleSyntax" : "isolatedModules";
+    // It is an error ro use `importsNotUsedAsValues` alongside `verbatimModuleSyntax`, but we still need to not crash.

```

Comment:
Shameful nit:

```suggestion
    // It is an error to use `importsNotUsedAsValues` alongside `verbatimModuleSyntax`, but we still need to not crash.
```

Reply:
https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FTypeScript%2Fpull%2F53383%23pullrequestreview-1349124507&data=05%7C01%7CJacob.Bailey%40microsoft.com%7C22bfbfc87d50418fcd2208db296c39f5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638149317029984634%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=J3Cfl5%2BXXEWqTJNep4aVpRuqUFC%2FdAjuXR2QHZVaTmM%3D&reserved=0

Reply:
(removed "shameful", i meant shameful that I'd send a 1 character diff, but I realized after I sent it that it sounded like I was shaming the typo which is absolutely NOT what I meant, sorry)